pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -385,31 +399,41 @@ const MemoryRow = ({ memorySize, memorySizeUnit, nodeMaxMemory, recommendedMemor
-                    initialUnit={memorySizeUnit}
-                    onValueChange={e => onValueChanged('memorySize', e.target.value)}
-                    onUnitChange={value => onValueChanged('memorySizeUnit', value)} />
-                <HelpBlock id="memory-size-helpblock">
-                    {validationStateMemory === "error" && <p>{validationFailed.memory}</p>}
-                    {recommendedMemoryHelpBlock}
-                    {nodeMaxMemory && <p> {cockpit.format(
-                        _("Up to $0 $1 available on the host"),
-                        Math.floor(convertToUnit(nodeMaxMemory, units.KiB, memorySizeUnit)),
-                        memorySizeUnit,
-                    )}</p>}
-                </HelpBlock>
-            </FormGroup>
+            <Expandable id='memory-details'
+                    toggleText={expandedMemory ? 'Hide Memory Details' : 'Show Memory Details'}
+                    onToggle={() => onValueChanged('expandedMemory', !expandedMemory)}
+                    isExpanded={expandedMemory}
+                    className='control-label'>
+                <span />
+            </Expandable>
+            <span />
+            {expandedMemory && <>
+                <label htmlFor='memory-size' className='control-label'>
+                    {_("Memory")}
+                </label>
+                <FormGroup validationState={validationStateMemory} bsClass='form-group ct-validation-wrapper' controlId='memory'>
+                    <MemorySelectRow id='memory-size'
+                        value={memorySize}
+                        maxValue={nodeMaxMemory && convertToUnit(nodeMaxMemory, units.KiB, memorySizeUnit)}
+                        initialUnit={memorySizeUnit}
+                        onValueChange={e => onValueChanged('memorySize', e.target.value)}
+                        onUnitChange={value => onValueChanged('memorySizeUnit', value)} />
+                    <HelpBlock id="memory-size-helpblock">
+                        {validationStateMemory === "error" && <p>{validationFailed.memory}</p>}
+                        {recommendedMemoryHelpBlock}
+                        {nodeMaxMemory && <p> {cockpit.format(
+                            _("Up to $0 $1 available on the host"),
+                            Math.floor(convertToUnit(nodeMaxMemory, units.KiB, memorySizeUnit)),
+                            memorySizeUnit,
+                        )}</p>}
+                    </HelpBlock>
+                </FormGroup>
+            </>}
         </>
     );
 };
 
-const StorageRow = ({ connectionName, storageSize, storageSizeUnit, onValueChanged, recommendedStorage, storagePoolName, storagePools, storageVolume, vms, validationFailed }) => {
+const StorageRow = ({ connectionName, storageSize, storageSizeUnit, onValueChanged, recommendedStorage, storagePoolName, storagePools, storageVolume, vms, expandedStorage, configureStorage, validationFailed }) => {
```

Comment:
>     1. Unattended installation isn't supported, yet it's still here in the UI and it isn't disabled. It just has a hover tooltip and you cannot click the checkbox.

It's not that it isn't supported - but one should first choose the OS and then the unattended installation checkbox will be clickable.
That's because there is content that will appear - the profiles selection - that are directly associated with the OS choice.
So I really prefer to not hide it and show it again once the user selected an OS since always it's going to appear.

> 
>     2. Expanders have an odd grey background and they also do not have a focus state. (Perhaps this is related?)
> 
> 
> ![Screenshot_2019-09-17 Virtual Machines - Rain](https://user-images.githubusercontent.com/10246/65034178-cbb3b100-d946-11e9-89a0-f2531ee0572b.png)